resources/leiningen/new/dw_code_exercise/home.clj

```diff
@@ -59,10 +59,19 @@
      [:li [:code "ocd-division/country:us/state:al/place:birmingham"]]]
     [:li "Not all of those are derivable from just an address (without running it through a standardization and augmentation service). "
      "For example, just having a random address in Birmingham doesn't tell us what county it is in. "
-     "But we can derive a basic set of country, state, and place OCD-IDs that will be a good starting point for this project."]
+     "But we can derive a basic set of country, state, and place OCD-IDs that will be a good starting point for this project. "
```

Comment:
This suggests using the country OCD-ID, but it's not in the instructions or the example `curl` request. Let's either remove it here, or make sure it appears in both places below.

Reply:
Yeah, I went back and forth on that. It doesn't hurt to include it, it will just never match anything. But it is a potential source of confusion so I'll delete it.